src/konserve/impl/defaults.cljc

```diff
@@ -97,23 +97,33 @@
 
         (when (:sync-blob? config)
           (trace "syncing for " key)
-          (<?- (-sync new-blob env))
-          (<?- (-sync-store backing env)))
+          (<?- (-sync new-blob env)))
         (<?- (-close new-blob env))
 
         (when-not (:in-place? config)
           (trace "moving blob: " key)
-          (<?- (-atomic-move backing new-store-key store-key env)))
+          (<?- (-atomic-move backing new-store-key store-key env))) 
+
+        (when (:sync-blob? config)
```

Comment:
This is an important change to fsync the directory after the move to ensure that the new file is visible.